src/AzureStack/GalleryAdmin/Gallery.Admin.Tests/Gallery.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Gallery.Admin.Tests</PackageId>
+    <Description>Gallery.Admin.Tests Class Library</Description>
+    <AssemblyName>Gallery.Admin.Tests</AssemblyName>
+    <VersionPrefix>1.0.0</VersionPrefix>    
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1.2</TargetFrameworks>
```

Comment:
@deathly809 any reason you need these additional frameworks

Reply:
Fixed.